### PR TITLE
Fix widths of table columns

### PR DESCRIPTION
**Pull request by siemiatj**
Header `th` elements also need to have widths for their classes defined. Relates to #1679 

Huge cleanup of table-specific styles. @PabloSichert please take a look if I haven't missed anything obvious :)

### CODE REVIEW

File: src/assets/css/table.css [hunk 1]
```diff
@@ -1,119 +1,183 @@
...
+        }
+    }
+
+    /* Table cols sizing */
+    .td-lg {
+        width: 225px;
+        max-width: 225px;
+        min-width: 225px;
+
+        .input-dropdown-list {
```
Comment (siemiatj) [inline src/assets/css/table.css]: @PabloSichert you recently removed the rule that was setting this width to 225px, but there was also another one that set it to auto. Does this collide with your changes ?

File: src/components/table/Table.js [hunk 2]
```diff
@@ -953,87 +953,86 @@ class Table extends Component {
...
+            indentSupported,
+            selected,
+            docId,
+            tabIndex,
+            readonly,
+            collapsible,
+            viewId,
+          }}
+          key={`${i}-${docId}`}
+          collapsed={collapsedParentsRows.indexOf(item[keyProperty]) > -1}
```
Comment (PabloSichert) [inline src/components/table/Table.js]: For readability, you could replace all

```js
···.indexOf(···) > -1
```

with

```js
···.includes(···)
```

File: src/assets/css/table.css (hunk 1 above)
Comment (PabloSichert) [reply src/assets/css/table.css]: I didn't test for functionality there. Just randomly discovered and removed an obsolete rule that would never apply. It was something like

```css
.x .y {
  width: 123px;
}

.x .y {
  width: auto;
}
```

File: src/assets/css/table.css [hunk 3]
```diff
@@ -1,119 +1,183 @@
...
+    &.table-bordered-vertically .item-caption td {
+        border-right: none;
+    }
 
-.table.table-bordered-horizontally td {
-    border-bottom: 1px solid $brand-border-color;
-}
+    &.table-bordered-vertically td.indented {
+        line-height: 28px;
+        min-width:0;
```
Comment (PabloSichert) [inline src/assets/css/table.css]: Just a small nitpick, there are some spaces missing here. You can leave them like this until we introduce a tool to automatically fix CSS formatting

File: src/assets/css/table.css (hunk 1 above)
Comment (siemiatj) [reply src/assets/css/table.css]: Yeah, it was a dupe. Ok.

**Comment by siemiatj** [comment]
No idea what is Codacy trying to do here, but none of this errors that it's showing is an error.

**Comment by metas-mk** [comment]
Checked locally. Fixed. Checked on 1980x1200/1080. Works. Merging.

**Pull request merged**